### ISSUE TITLE
Standalone ESXi compatability

### ISSUE THREAD

**Issue by chayes38** [jorgedlcruz/vmware-grafana#12]
Does this work with standalone ESXi servers.  I have a fully licensed ESXi instance and am successfully gathering "VMware vSphere - Hosts" & "- Overview" statistics. However "- VMs" & "-Datastore" graphs are empty.  

If this is the wrong forum, I apologize.  I'm a newbie and just getting my feet wet.

Chris

**Comment by jorgedlcruz** [comment]
Hello,
No, this project works only with vSphere vCenter SDK, and the Standalone ESXi doesn't have it. I am working on an SNMP project, which will work with both, ESXi Free or Licensed, but standalone and not need of vCenter, but for now this project relies on the vCenter SDK I am afraid.

**Comment by chayes38** [comment]
Thanks for your quick response.  Looking forward to the SNMP version.

**Comment by aofvalley** [comment]
First of all, @jorgedlcruz thanks for this awesome work!!

I have the same issue, I'm waiting for the SNMP Version, Vsphere its expensive for me and use the Free Version :(

ESP: Gracias Jorge, te sigo por tu blog y es placer aprender de ti!! Esperando la version SNMP me hallo, un saludo!

**Comment by yurividal** [comment]
Yes, indeed. I also follow @jorgedlcruz on his blog, and my team and I are anxiously waiting for this SNMP implementation, it would be pretty awesome.

**Comment by oijkn** [comment]
+1

If you wan't I can help you in this project (y hablo también español).

**Comment by oijkn** [comment]
Hi, any good news, please ?

**Comment by ronnyandre** [comment]
Would be awesome if there were some progress on this!
